library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -1358,6 +1357,24 @@ public void setOnRangeSelectedListener(OnRangeSelectedListener listener) {
     }
 
     /**
+     * Add listener to the title.
+     *
+     * @param listener thing to be notified
+     */
+    public void setOnTitleClickListener(final OnClickListener listener) {
+        if (listener != null) {
+            title.setOnClickListener(listener);
```

Comment:
What's wrong with setting the listener to null if it should already be guarding for null listeners